frontend/src/app/main/ui/auth/register.cljs

```diff
@@ -160,13 +161,6 @@
 (defn- handle-register-error
   [form error]
   (case (:code error)
-    :registration-disabled
-    (st/emit! (dm/error (tr "errors.registration-disabled")))
-
-    :email-has-permanent-bounces
-    (let [email (get @form [:data :email])]
-      (st/emit! (dm/error (tr "errors.email-has-permanent-bounces" email))))
-
```

Comment:
Why are those removed?

Reply:
because they are not handled here, so this is technically dead code.